crypto/rand/drbg_lib.c

```diff
@@ -554,7 +554,9 @@ int rand_drbg_restart(RAND_DRBG *drbg,
             drbg->meth->reseed(drbg, adin, adinlen, NULL, 0);
         } else if (reseeded == 0) {
             /* do a full reseeding if it has not been done yet above */
-            RAND_DRBG_reseed(drbg, NULL, 0, 0);
+            if(!RAND_DRBG_reseed(drbg, NULL, 0, 0)) {
```

Comment:
There's a formatting nit here. Should be `if (`, i.e. with a space after `if` and before opening parens. Is this already like this on master? If so then possibly leave it.

Reply:
`drbg_lib.c` no longer exists on master.